src/HDR/HdrCache.cpp

```
@@ -0,0 +1,403 @@
+}
+
+bool writeHeader(QFile& file, uint32_t payloadBytes)
+{
+    const FileHeader header{kMagic, kFormatVersion, payloadBytes};
+    return writeExact(file, &header, sizeof(FileHeader));
+}
+
+bool readCubemapFaces(QFile& file, HdrEquirect::CubemapFaces& faces, QString& error)
+{
+    int32_t faceSize = 0;
+    if (!readExact(file, &faceSize, sizeof(int32_t)) || faceSize <= 0) {
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_DECLARE_METATYPE is a macro then please configure it.

(unknownMacro)